google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/models/RowMutationTest.java

```diff
@@ -90,6 +90,32 @@ public void toBulkProtoTest() {
         .isIn(timestampRange);
   }
 
+  @Test
+  public void toProtoTestWithProvidedMutation() {
+    long timestampMin = System.currentTimeMillis() * 1_000;
+
+    Mutation mutation = Mutation.create().setCell("fake-family", "fake-qualifier", "fake-value");
+    RowMutation rowMutation = RowMutation.create("fake-table", "fake-key", mutation);
+
+    MutateRowsRequest actualRowMutation = rowMutation.toBulkProto(REQUEST_CONTEXT);
+
+    com.google.common.collect.Range<Long> timestampRange =
+        com.google.common.collect.Range.closed(timestampMin, System.currentTimeMillis() * 1_000);
```

Comment:
Can you just explicitly set the timestamp and avoid the complexity of timestamp ranges?